pkg/vars/evaluator.go

```diff
@@ -33,6 +34,22 @@ func (fp ResponseMessageEvaluator) Eval(req *mock.Request, m *mock.Definition) {
 	fp.walkAndFill(m, vars)
 }
 
+func (fp ResponseMessageEvaluator) EvalCallback(req *mock.Request, m *mock.Definition) {
```

Comment:
Instead of a new function, why you don't just add:

This line: `fp.extractVars(m.Callback.Body, &holders)` to `func Eval` and
and this: `m.Callback.Body = fp.replaceVars(m.Callback.Body, vars)` to `func walkAndFill` then you can remove this line https://github.com/jmartin82/mmock/pull/111/files#diff-afe66be6a9c042e9570b60e75acbc5cfR100 and you will avoid code repetition (error-prone) and that the extra fillers allocations

Reply:
Much simpler, thanks!

Reply:
Nice, updated :)